service/frontend/adminHandler.go

```diff
@@ -1584,19 +1599,14 @@ func (adh *AdminHandler) GetTaskQueueTasks(
 		return nil, adh.error(err, scope)
 	}
 
-	taskMgr := adh.taskManager
-
-	maxTaskID := request.GetMaxTaskId()
-	req := &persistence.GetTasksRequest{
+	resp, err := adh.taskManager.GetTasks(&persistence.GetTasksRequest{
 		NamespaceID:        namespaceID.String(),
 		TaskQueue:          request.GetTaskQueue(),
 		TaskType:           request.GetTaskQueueType(),
 		MinTaskIDExclusive: request.GetMinTaskId(),
-		MaxTaskIDInclusive: maxTaskID,
+		MaxTaskIDInclusive: request.GetMaxTaskId(),
```

Comment:
why is this minExclusive and maxInclusive?

Reply:
Matching persistence APIs still have this behavior. Let me create a task for it.

Reply:
https://github.com/temporalio/temporal/issues/2587